dev-server/templates/index.mustache

```diff
@@ -38,6 +38,7 @@
           configuration option enabled</a></li>
       <li><a href="/document/doyle-tiny"><i>The Disappearance of Lady Carfax</i> with tiny (10px) root font size in host page</a></li>
       <li><a href="/document/doyle-huge"><i>The Disappearance of Lady Carfax</i> with huge (24px) root font size in host page</a></li>
+      <li><a href="/document/doyle-centered"><i>The Disappearance of Lady Carfax</i> with a centered, max-width body</a></li>
```

Comment:
I think we could shorten the book title here, as the content itself isn't important, just the styling effect. Maybe just to "Book with <styling effect>"?